agent/engine/docker_task_engine_test.go

```diff
@@ -330,40 +315,39 @@ func TestRemoveEvents(t *testing.T) {
 			ExitCode: &exitCode,
 		},
 	}
+	steadyStateVerify <- time.Now()
```

Comment:
I think this line is essentially a nop as nothing is waiting on this channel by the time we get here. I'll get rid of it.